README.md

```diff
@@ -55,13 +57,33 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
+If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests.
+
+`NOTE: Commands are to be executed on Linux, Mac, and Windows(using Powershell)`
+
+1. Download the Mattermost docs repo onto a machine with Python installed or simple cloned the forked copy of this repo by running,
+```sh
+$ git clone https://github.com/mattermost/docs.git
+```
+2. Install [pipenv](https://docs.pipenv.org/) by following command
+```sh
+$ brew install pipenv   # for Mac users, if you have brew installed)
```

Comment:
```suggestion
$ brew install pipenv (macOS)
```

Reply:
I think of overriding this suggestion and keeping the change, as this will help people to gain more clarity on what commands.

Reply:
Thanks, @helios1101! That's a good point. How about something like: "(if you don't have Homebrew installed, download it here") or similar?